src/components/common/SimpleTable/index.tsx

```diff
@@ -89,7 +89,6 @@ const Wrapper = styled.table<{ $numColumns?: number }>`
 
     > th, 
     > td {
-      height: 3rem;
```

Comment:
Remove the height from the pure component. Offload specific height requirements to whatever component.